### PR TITLE
Add Categories title for Themes categories section

### PR DESCRIPTION
**Pull request by atiqueahmedziad**
Fixes issue #2272 
### Before
screenshot from 2016-07-01 15 37 35
### After
screenshot from 2016-07-01 15 36 19

### CODE REVIEW

**Comment by atiqueahmedziad** [comment]
@tofumatt , r?

File: static/css/zamboni/zamboni.css [hunk 1]
```diff
@@ -2405,6 +2404,10 @@ form .error .note.error {
     font-size: 1.05em;
 }
 
+.other-categories.highlight ul:nth-of-type(1) {
```
Comment (muffinresearch) [inline static/css/zamboni/zamboni.css]: Is there a need to only do this for the first one?

As a side-note using `:first-of-type` might be a more direct way to do the same thing.

File: src/olympia/browse/templates/browse/macros.html [hunk 2]
```diff
@@ -11,13 +11,14 @@
...
-    <ul>
-      {% for cat in categories %}
-        <li id="p-{{ cat.id }}">
-          <a href="{{ profile.get_user_url('themes', args=[cat.slug]) if profile else url(viewname, cat.slug) }}">
-            {{ cat.name }}</a>
-        </li>
-      {% endfor %}
-    </ul>
+    <h3>Categories</h3>
+      <ul>
```
Comment (muffinresearch) [inline src/olympia/browse/templates/browse/macros.html]: The `ul` doesn't doesn't need to be indented.

File: src/olympia/browse/templates/browse/macros.html [hunk 3]
```diff
@@ -11,13 +11,14 @@
...
+        {% for cat in categories %}
+          <li id="p-{{ cat.id }}">
+            <a href="{{ profile.get_user_url('themes', args=[cat.slug]) if profile else url(viewname, cat.slug) }}">
+              {{ cat.name }}</a>
```
Comment (tofumatt) [inline src/olympia/browse/templates/browse/macros.html]: I'd put the `</a>` here on the next line.

File: src/olympia/browse/templates/browse/macros.html [hunk 4]
```diff
@@ -11,13 +11,14 @@
...
     {% endif %}
-    <ul>
-      {% for cat in categories %}
-        <li id="p-{{ cat.id }}">
-          <a href="{{ profile.get_user_url('themes', args=[cat.slug]) if profile else url(viewname, cat.slug) }}">
-            {{ cat.name }}</a>
-        </li>
-      {% endfor %}
-    </ul>
+    <h3>Categories</h3>
```
Comment (tofumatt) [inline src/olympia/browse/templates/browse/macros.html]: Please use gettext so this can be translated, eg:

`{{ _('Categories') }}`

**Comment by atiqueahmedziad** [comment]
@muffinresearch & @tofumatt thanks for the suggestions.
Done as per your suggestions.

**Comment by atiqueahmedziad** [comment]
test passed. :smile:

File: src/olympia/browse/templates/browse/macros.html [hunk 5]
```diff
@@ -3,21 +3,21 @@
...
+     <ul>
         {% for key, title_ in filter.opts_dict.items() %}
           <li id="p-{{ key }}">
             <a href="{{ base_url|urlparams(sort=key) }}">{{ title_ }}</a>
           </li>
         {% endfor %}
-      </ul>
     {% endif %}
-    <ul>
+      <h3>{{ _('Categories') }}</h3>
```
Comment (muffinresearch) [inline src/olympia/browse/templates/browse/macros.html]: You can't stick a `h3` in the middle of a `ul` like this. You'll need to re-instate the closing an opening `ul`

File: src/olympia/browse/templates/browse/macros.html [hunk 6]
```diff
@@ -3,21 +3,21 @@
   <div class="other-categories highlight">
     <h3><a href="{{ base_url }}">{{ title }}</a></h3>
     {% if filter %}
-      <ul>
```
Comment (muffinresearch) [inline src/olympia/browse/templates/browse/macros.html]: This should be indented as it was before.

**Comment by muffinresearch** [comment]
> test passed.

Fwiw style and markup changes are not generally tested in this repo (something that will change in the future), so it's important to note - don't rely on tests passing to mean anything for this kind of change :)

File: src/olympia/browse/templates/browse/macros.html (hunk 6 above)
Comment (tofumatt) [inline src/olympia/browse/templates/browse/macros.html]: Hey @atiqueahmedziad, can you fix the indentation here? Like @muffinresearch said, it should be the way it already was.

File: src/olympia/browse/templates/browse/macros.html [hunk 7]
```diff
@@ -3,21 +3,21 @@
...
       {% for cat in categories %}
         <li id="p-{{ cat.id }}">
           <a href="{{ profile.get_user_url('themes', args=[cat.slug]) if profile else url(viewname, cat.slug) }}">
-            {{ cat.name }}</a>
+            {{ cat.name }}
+          </a>
         </li>
       {% endfor %}
-    </ul>
+     </ul>
```
Comment (tofumatt) [inline src/olympia/browse/templates/browse/macros.html]: Same here, this indentation shouldn't be changed.

**Comment by atiqueahmedziad** [comment]
sorry, i thought a wrong meaning of "indented". 
I hope, its okey, this time :)

**Comment by tofumatt** [comment]
r+

Sorry I just missed the cutoff for this week's push as I have been distracted all day in my new apartment!

This should make it into production next week though, thanks for your patch and patience with the review, @atiqueahmedziad!

**Comment by atiqueahmedziad** [comment]
no problem :) anyways thanks for helping :smile:

**Pull request merged**